design/ceph/external-mon-arbiter.md

```diff
@@ -0,0 +1,60 @@
+---
+title: external Mon arbiter
+target-version: release-1.17
```

Comment:
Is this feature still targeted for 1.17? #15400 could potentially be backported to 1.16, but perhaps better to let it stabilize in master first? I could go either way if we feel confident in the testing.

Reply:
For me both options are ok. Just one question:

Is it makes sense to just close this PR? 
Implementation PR now also contains almost the same documentation. Probably in this case it is better to have doc in one place.

Reply:
True, if we pretty much have the content in the docs with the other PR, we can close this one.